lib/auth_plug.ex

```diff
@@ -128,7 +116,7 @@ defmodule AuthPlug do
   defp get_token_from_header({"authorization", value}) do
     value = String.replace(value, "Bearer", "") |> String.trim()
     # fast check for JWT format validity before slower verify:
-    if is_nil(value) do
+    if is_nil(value) do # Does this ever eval to true?
```

Comment:
yeah, for some reason when there is no value in the header it behaves unpredictably hence setting it to nil here. 🤷